_includes/home/about.html

```diff
@@ -1,32 +1,38 @@
-<section class="about">
+<section class="about" id="about">
   <div class="container">
     <div class="row">
       <div class="col-lg-12 text-center">
-        <h2 class="section-heading">About Us</h2>
-        <h3 class="section-subheading text-muted">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Minima maxime quam architecto quo inventore harum ex magni, dicta impedit.</h3>
+        <h2 class="section-heading">About TeachOSM</h2>
+        <div class="section-subheading text-muted">TeachOSM was chartered in 2016 as a project of OpenStreetMap US (OSM US) to provide operational support and strategic guidance for the educational outreach mission of OSM US.
+
+        The TeachOSM website is an online resource to assist educators at all levels to introduce open source mapping using the OpenStreetMap ecosystem of tools in the classroom. Individual instructors are encouraged to make use of the instructional materials provided here to develop assignments for their particular discipline and curricular needs. This site provides the resources help instructors identify, assign, manage and grade a mapping assignment. This site also includes training documentation for both the instructor and students.
+
+        TeachOSM is led by a volunteer steering committee based throughout the United States, and relies upon teachers, students, and OpenStreetMappers all over the globe.
+        </div>
+        </br></br>
       </div>
     </div>
     <div class="row text-center">
       <div class="col-md-4">
         <span class="fa-stack fa-4x">
           <i class="fa fa-newspaper-o fa-stack-1x fa-inverse"></i>
         </span>
-        <h4 class="about-heading">Lesson Plans</h4>
-        <p class="text-muted">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Minima maxime quam architecto quo inventore harum ex magni, dicta impedit.</p>
+        <h4 class="EducationalContent-heading">Educational Content</h4>
+        <p class="text-muted">The Educational Portal is where educators can share educational exercises, projects and lesson plans with one another and obtain contact information for educators around the world.</p>
       </div>
       <div class="col-md-4">
         <span class="fa-stack fa-4x">
           <i class="fa fa-map-o fa-stack-1x fa-inverse"></i>
         </span>
-        <h4 class="service-heading">Resources</h4>
-        <p class="text-muted">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Minima maxime quam architecto quo inventore harum ex magni, dicta impedit.</p>
+        <h4 class="EducatorTraining-heading">Educator Training</h4>
```

Comment:
Are these classes being used for anything? I couldn't find any associated css. I recommend that we don't add any unused classes.

Reply:
ok will remove the unused classes